src/constants/index.js

```diff
@@ -0,0 +1,2 @@
+export const ROOT_URL = 'http://localhost:5000';
+// export const ROOT_URL = 'https://yummy-recipe-api.herokuapp.com';
```

Comment:
In production, environment variables should be used so you can have a single line pick from the env variable